components/function-controller/cmd/manager/main.go

```diff
@@ -20,77 +20,82 @@ import (
 	"flag"
 	"os"
 
-	buildv1alpha1 "github.com/knative/build/pkg/apis/build/v1alpha1"
-	servingv1alpha1 "github.com/knative/serving/pkg/apis/serving/v1alpha1"
-	"github.com/kyma-project/kyma/components/function-controller/pkg/apis"
-	"github.com/kyma-project/kyma/components/function-controller/pkg/controller"
-	"github.com/kyma-project/kyma/components/function-controller/pkg/webhook"
+	// allow client authentication against GKE clusters
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+
+	"k8s.io/apimachinery/pkg/runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+
+	buildv1alpha1 "github.com/knative/build/pkg/apis/build/v1alpha1"
+	servingv1alpha1 "github.com/knative/serving/pkg/apis/serving/v1alpha1"
+
```

Comment:
empty line, do we need to run goimport?

Reply:
It's perfectly valid as per goimports. Separating imports is good I believe, it helps showing what is internal, what is Knative, etc.